### DISCUSSION TITLE
Suggestion: Cool Image Slideshow

### DISCUSSION THREAD

**Discussion by zencyon** [d2phap/ImageGlass#1450] [General]
It seems slideshows of images slowly animating, zooming, panning and fading into the next in cool ways are a thing of the past
I would suggest options for cool animated slideshows! :)

nearly all image viewers seem to default to standard image by image and a timer, that's it, we can do better I think
once upon a time windows had such a screensaver, alas, M$ put an end to it, as they do

ImageGlass is a great viewer, I think this would make it even cooler :) cheers!

**Comment by d2phap** [comment]
ImageGlass 8 does not have any transitions for the slideshow at all.

In v9.0 beta 3, I have decoupled slideshow function into a separated window/process, which allows to have multiple slideshow windows and creates space for transitions.

Also, I added support for fading transition.
